src/EpicWithHeaderImage/index.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { css } from '@emotion/react';
+import { from, breakpoints } from '@guardian/src-foundations/mq';
+import { space } from '@guardian/src-foundations';
+import { body, headline } from '@guardian/src-foundations/typography';
+
+import { Epic } from '../Epic';
+import type { EpicProps } from '../Epic';
+
+export { COMPONENT_NAME } from './canRender';
+
+const HEADER_IMAGE_URL =
+    'https://i.guim.co.uk/img/media/cecfef4098a2a8c1e302e3f67b979f11ee529bb6/0_0_470_471/master/470.png?width=300&quality=45&s=d654e72595c07e2095777863f4901863';
+
+const emptyImage = 'data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw==';
+
+const headerStyles = {
+    picture: css``,
+    rightContainer: css`
+        flex: 1;
+    `,
+    image: css`
+        height: auto;
+        width: 100%;
+        object-fit: cover;
+    `,
+    container: css`
+        display: none;
+
+        ${from.tablet} {
+            display: flex;
+            align-items: center;
+            padding: ${space[1]}px ${space[1]}px 0;
+        }
+    `,
+    leftContainer: css`
+        flex: 3;
+        margin-right: ${space[4]}px;
+    `,
+    text: css`
+        ${headline.large({ fontWeight: 'bold' })};
+        line-height: 1.35;
+    `,
+    imageCaptionContainer: css``,
+    imageCaption: css`
+        ${body.medium()}
+        margin: 0;
+    `,
+    imageCaptionBold: css`
+        ${body.medium({ fontWeight: 'bold' })}
+    `,
+    imageCaptionItalic: css`
+        ${body.medium({ fontStyle: 'italic' })}
+    `,
+};
+
+// We use a picture element here with an empty image for < tablet so that we
+// don't load an image unnecessarily when we're not going to render it (which would be
+// the case with an img tag)
+const HeaderSection: React.FC = () => (
+    <div css={headerStyles.container}>
+        <div css={headerStyles.leftContainer}>
+            <span css={headerStyles.text}>
+                You&#8217;re powering open,&nbsp;independent journalism
```

Comment:
I used a non-breaking space here to keep `open, independent` on the same line. Interested in thoughts on that. It seems to have the desired effect on the browsers/widths I've tested on.

Reply:
Sounds good if we've checked it doesn't cause problems at any width. A more conservative approach in general is to add word breaks elsewhere. This mean the browser **favours** other splits rather than **forcing** this lack of split, which gives it the flexibility to make this break if doing so is completely necessary.

If this isn't being shown at mobile widths then I imagine this is fine!

Reply:
Also a good point 👍 

> is to add word breaks elsewhere

You mean with `<wbr>`? I've not used those much before, I'll take a look.

I just tried removing the nbsp and the browser is wrapping things exactly the same, maybe this was making a difference in a previous version where this top section was laid out slightly differently. So I'm tempted to remove it. (In a future where that text is dynamically provided we probably wouldn't have that level of control, so maybe better to let the browser get on with it).

Reply:
Yeah a `<wbr>` tag is one way, IIRC there's an entity for it as well*. It's well-supported in theory but I don't know how strongly browsers would stick to it these days. My experience with this is from a few years back!

* Like me, this page is very old but has some information on the options
    https://www.quirksmode.org/oddsandends/wbr.html

No dramas though, it was fine as-is and even better if it is working without.

Reply:
Thanks for this! Ahh quirksmode, I haven't thought about that site for ages. I used to refer to it all the time.